cmd/amppkg/main.go

```diff
@@ -127,7 +127,7 @@ func main() {
 	}
 
 	packager, err := signer.New(certs[0], key, config.URLSet, rtvCache, certCache.IsHealthy,
-		overrideBaseURL, /*requireHeaders=*/!*flagDevelopment && !*flagInvalidCert)
+		overrideBaseURL, /*requireHeaders=*/!(*flagDevelopment || *flagInvalidCert))
```

Comment:
Actually, on further consideration, are you sure `flagInvalidCert` belongs here? `requireHeaders` seems unrelated to the invalid cert:
https://github.com/ampproject/amppackager/blob/e4bf0430ba152cfe82ccf063df92021dfc0f26a5/packager/signer/signer.go#L352-L362

For testing one can use `curl`, `amppkg_dl_sxg`, or something like [ModHeader](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj) or [Requestly](https://www.requestly.in/).

Reply:
Thanks for looking at this closer. You're correct. The `-invalidcert` flag is meant to be "otherwise production", so yes, we should expect a valid `AMP-Cache-Transform` header.

Reply:
I verified commit 97f8a38 works as intended when sending values `nothing` and `any` for header `AMP-Cache-Transform`.